pkgs/development/libraries/ndn-cpp/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, openssl, doxygen, zlib, pkgconfig, protobuf }:
+let
+  version = "v0.15";
+in
+stdenv.mkDerivation {
+  name = "ndn-cpp-${version}";
```

Comment:
The new way to set the name is like so:
```
pname = "ndn-cpp";
version = "v0.15";
```
With this `name` will automatically be set to `ndn-cpp-v0.15`.